File: Arreglos.java
import java.util.Scanner;
public class Arreglos {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int[] numeros = new int[6];
		
		//capturar los numeros denntro del arreglo
		for (int i = 0; i < numeros.length; i++) {
			System.out.println("ingrese un numero");
			numeros[i] = sc.nextInt();
		}
		//mostrar los numeros dentro del arreglo
		for (int i = 0; i < numeros.length; i++){
			System.out.println("el numero en la posicion " + i + " es: " + numeros[i]);
		}
		
	}
}

File: arreglos.cpp
#include <iostream>
using namespace std;

int main(){
    string nombres [5];
    //llenar el arreglo
    for(int i = 0; i < 5; i++){
        cout << "ingresa un nombre:";
        cin >> nombres[i];
    }
    //mostrar los datos del arreglo
    for (int i = 0; i < 5; i++){
        cout << nombres[i];
    }
    return 0;

}
